METRICS.md

```diff
@@ -3,35 +3,55 @@ This documents the metrics and tags emitted by gostatsd, their type, tags, and i
 | Name                         | type    | tags            | description
 | ---------------------------- | ------- | --------------- | -----------
 | aggregator.metrics_received  | gauge   | aggregator_id   | The number of datapoints received during the flush interval
-| aggregator.processing_time   | gauge   | aggregator_id   | The time taken (in ms) to aggregate all datapoints in this
-|                              |         |                 | flush interval
+| aggregator.aggregation_time  | gauge   | aggregator_id   | The time taken (in ms) to aggregate all counter and timer
+|                              |         |                 | datapoints in this flush interval
+| aggregator.process_time      | gauge   | aggregator_id   | The time taken to process all synchronous flush actions (aka the
+|                              |         |                 | blocking phase)
+| aggregator.reset_time        | gauge   | aggregator_id   | The time taken to reset the aggregator after flush
 | bad_lines_seen               | counter |                 | The number of unprocessable lines that have been seen
 | events_received              | counter |                 | The number of events received
 | metrics_received             | counter |                 | The number of metrics received
 | packets_received             | counter |                 | The number of packets received
 | avg_packets_in_batch         | gauge   |                 | The average number of packets read in a batch (up to receive-batch-size).
 |                              |         |                 | This can be used to tweak receive-batch-size if necessary to reduce memory usage
+| channel.avg                  | gauge   | channel         | The average of all samples
+| channel.min                  | gauge   | channel         | The minimum sample seen
+| channel.max                  | gauge   | channel         | The maximum sample seen
+| channel.last                 | gauge   | channel         | The last sample seen
 | channel.capacity             | gauge   | channel         | The capacity of the channel
-| channel.queued               | gauge   | channel         | The absolute amount of items in a channel
-| channel.pct_used             | gauge   | channel         | The percentage of how full a channel is
+| channel.samples              | gauge   | channel         | The number of samples seen
 | internal_dropped             | gauge   |                 | The number of internal metrics which have been dropped in the
 |                              |         |                 | lifetime of the process.  Not a counter, because it may not be
 |                              |         |                 | submitted.
```

Comment:
Yeeaah, as I understand it, markdown doesn't allow for manual word wrapping in a table (that I can find, anyway).  I opted for "readble as a text file, ugly in a render" instead of "not readable in a text file, pretty in a render".